cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableCore.java

```diff
@@ -106,55 +99,65 @@ public synchronized boolean swap(byte[] row, byte[] column, byte[] oldValue, byt
       // to-do
       deleteColumn(row, column);
     } else {
-      persist(Collections.singletonMap(row, Collections.singletonMap(column, newValue)), System.currentTimeMillis());
+      persist(Collections.singletonMap(row, Collections.singletonMap(column, newValue)), Long.MAX_VALUE);
```

Comment:
This method is in `LevelDBTableCore`, which is not specific for metrics, so we should not assume the version is no use. Should we introduce a new function which takes a version?

Reply:
👍 Should we extract this version as some Constant so that we can use it for all the places?

Reply:
It is only used in this class and why need to define a constant for another constant (Long.MAX_LONG is a constant)?

Reply:
It is also used in `LevelDBMetricsTable`. Just to make sure that we are using same value as the version. But it is just minor not a big deal.

Reply:
Technical they are independent and doesn’t have to be the same, right?